services/leaderboard/src/main/java/io/openliberty/spacerover/leaderboard/rest/Leaderboard.java

```diff
@@ -136,56 +140,21 @@ private String getJsonListFromIterableDocuments(FindIterable<Document> docs) {
 	@Operation(summary = "Clear the leaderboard entries from the database.")
 	@DELETE
 	@Path("/")
-	public Response clear() {
+	public Response clear(@DefaultValue("") @QueryParam("playerID") String playerID) {
 		MongoCollection<Document> collection = db.getCollection(LEADERBOARD_COLLECTION_NAME);
-		DeleteResult result = collection.deleteMany(new Document());
-		if (result.wasAcknowledged()) {
-			return Response.status(Response.Status.OK).build();
-		} else {
-			return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity("[\"Unable to clear leaderboard!\"]")
-					.build();
+		DeleteResult result;
+		if(playerID.isEmpty())
+		{
+			result = collection.deleteMany(new Document());
+		}
+		else
+		{
+			result = collection.deleteMany(eq("player", playerID));
```

Comment:
nit: make the spacing consistent with the rest of the code

```suggestion
		DeleteResult result;
		if (playerID.isEmpty()) {
			result = collection.deleteMany(new Document());
		} else {
			result = collection.deleteMany(eq("player", playerID));
```